packages/repack/src/logging/reporters/ConsoleReporter.ts

```diff
@@ -71,8 +72,14 @@ const FALLBACK_SYMBOLS: Record<LogType | 'progress', string> = {
 
 class InteractiveConsoleReporter implements Reporter {
   private requestBuffer: Record<string, Object> = {};
+  bar = new cliProgress.SingleBar({
+    format: 'Compiling {platform}: ' + '|{bar}|' + ' %{percentage}' + ' ',
```

Comment:
Make sure to handle the initial state. Right now when the progress is 0% there's label with the placeholder "Compiling **{platform}**". Please do not show this placeholder.
![CleanShot 2023-06-15 at 14 09 09@2x](https://github.com/callstack/repack/assets/63900941/08605da4-7953-47a9-9c84-2ac0ffe57ec8)